venusian/__init__.py

```diff
@@ -12,6 +11,9 @@
 
 ATTACH_ATTR = '__venusian_callbacks__'
 LIFTONLY_ATTR = '__venusian_liftonly_callbacks__'
+IMP_PY_SOURCE = 1
```

Comment:
Where are these magic numbers documented?

Reply:
@bertjwregeer , I took the values from the `imp` source code:
https://github.com/python/cpython/blob/master/Lib/imp.py#L35-L45

Having said that, in this case I think they could be any value.

Reply:
I don't think these should be hard coded. According to https://docs.python.org/3/library/imp.html?highlight=imp#imp.get_magic the value may be different for each Python version. It also says which value to use from ``importlib`` instead: https://docs.python.org/3/library/importlib.html#module-importlib.machinery I haven't figured out the full mapping yet.

Reply:
If we go Python 3 only, we can ignore the whole "source file" vs ".pyc" file issue entirely, because .pyc files are not stored in the same path as source files, and we would potentially want to import .pyc files.